examples/src/main/java/io/grpc/examples/massage/MassageClient.java

```diff
@@ -0,0 +1,127 @@
+ * notice, this list of conditions and the following disclaimer.
+ *    * Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following disclaimer
+ * in the documentation and/or other materials provided with the
+ * distribution.
+ *
+ *    * Neither the name of Google Inc. nor the names of its
+ * contributors may be used to endorse or promote products derived from
+ * this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.grpc.examples.massage;
+
+import io.grpc.Attributes;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.NameResolver;
+import io.grpc.ResolvedServerInfo;
+import io.grpc.grpclb.GrpclbLoadBalancerFactory;
+
+import java.net.InetSocketAddress;
+import java.net.URI;
+import java.util.Collections;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * A simple client that demonstrate GRPCLB protocol.
+ */
+public class MassageClient {
+  private static final Logger logger = Logger.getLogger(MassageClient.class.getName());
+
+  private static final NameResolver nameResolver = new NameResolver() {
+    @Override public String getServiceAuthority() {
+      return "helloworld";
+    }
+
+    @Override public void start(NameResolver.Listener listener) {
+      listener.onUpdate(
+          Collections.singletonList(new ResolvedServerInfo(
+              new InetSocketAddress("127.0.0.1", 50010), Attributes.EMPTY)),
+          Attributes.EMPTY);
+    }
+
+    @Override public void shutdown() {
+    }
+  };
+
+  private static final NameResolver.Factory nameResolverFactory = new NameResolver.Factory() {
+    @Override public NameResolver newNameResolver(URI targetUri, Attributes params) {
+      return nameResolver;
+    }
+  };
+
+  private final ManagedChannel channel;
+  private final MassageGrpc.MassageBlockingStub blockingStub;
+
+  MassageClient() {
+    channel = ManagedChannelBuilder.forTarget("helloworld")
+        .usePlaintext(true)
+        .nameResolverFactory(nameResolverFactory)
```

Comment:
Wouldn't a "real" usage not specify `nameResolverFactory` and just set the `loadBalancerFactory()`? It seems weird to have an example like this. Maybe wait for the example until we have a name resolver that makes use of it? Or have a message saying that this isn't fully ready yet?